nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -41,6 +41,11 @@ The following binary components are provided under the Apache Software License v
       in some artifacts (usually source distributions); but is always available
       from the source code management (SCM) system project uses.
 
+  (ASLv2) HiveMQ MQTT Client
```

Comment:
This is good but I think we need to add it to nifi-assembly/NOTICE as well.

Reply:
Thanks for NOTICING that! (pun intended, change added)